### ISSUE TITLE
`showGridMenu()` is triggered when we hit ENTER key on the inline filter

### ISSUE THREAD

**Issue by yazhiniHariharan3** [ghiscoding/Angular-Slickgrid#447]
### Your Environment
| Software                | Version(s) |
| -------------------- | -------- |
| Angular                  | 8.2.14   | 
| Angular-Slickgrid   | 2.17.11 | 
| TypeScript              | 3.5.3     | 

### Describe the Bug
`showGridMenu()` is getting triggered when the user hits ENTER key after entering the search term on the compoundInputNumberFilter

### Steps to Reproduce
1. Enable filtering and use Filters.compoundInputNumber on a field.
2. Set the gridmenu with the below configuration.
3. In a compoundInputNumber field, type your search term and hit enter.
Current setup gridMenu Configurations
```
this.gridOptions = {
      gridMenu: {
        hideForceFitButton: true,
        columnTitle: '',
        hideRefreshDatasetCommand: true,
        iconCssClass: 'fa fa-bars'
      }
//Other options for filters
```
### Expected Behavior
The grid menu shouldn't show in the screen at all, when the user hits enter key after entering a search term for inline filtering.

### Current Behavior
When we hit enter key in the inline filtering, the library is implicitly invoking the showGridMenu() and the gridMenu shows up on Left side of the screen. The shown menu is ONLY partially visible.
![image](https://user-images.githubusercontent.com/18120900/81012876-d1182a00-8e49-11ea-87f9-f9246ed3744f.png)

**Comment by ghiscoding** [comment]
I can't reproduce this, I don't think any of the demos are showing this behavior. Also the `showGridMenu` is a method on the Grid Menu plugin and it's not really an event, so why are you using it as an event? So you most probably have some extra code that you did not include in this issue.

**Comment by yazhiniHariharan3** [comment]
Thanks for your swift reply. This is reproducible in your demo page as well. 
If you focus on the Grid menu like in the below screenshot and hit enter i n the keyboard, you should be able to see the grid menu opening on the left side of the screen.
![image](https://user-images.githubusercontent.com/18120900/81256294-adbdcc80-904d-11ea-9560-5285e8117bd8.png)
In our application, when we focus out of a cell, somehow the focus was going on to the grid menu and on hitting enter, it used to open the grid menu on the Left side of the screen. 
Thanks much !

**Comment by ghiscoding** [comment]
hmm this Grid Menu was never meant to be used from the keyboard. This is an issue with [SlickGrid](https://6pac.github.io/SlickGrid) (core) lib, it can be reproduced in this SlickGrid [Example](https://6pac.github.io/SlickGrid/examples/example-grid-menu.html).

Can you please open an issue in the core lib [SlickGrid](https://6pac.github.io/SlickGrid) instead. There's nothing to fix in Angular-Slickgrid, it will have to be fixed in the core lib. Once it's fixed in the core, it will flow to Angular-Slickgrid.

**Comment by yazhiniHariharan3** [comment]
Thanks much for having looked into this. Sure, I will raise a bug in SlickGrid library.

**Comment by ghiscoding** [comment]
Thanks, I'll close this issue here then

**Comment by ghiscoding** [comment]
This issue will be fixed in the core lib by this [PR #491](https://github.com/6pac/SlickGrid/pull/491)

**Comment by ghiscoding** [comment]
This is now available in latest version [2.18.x](https://github.com/ghiscoding/Angular-Slickgrid/releases). 

Please upvote ⭐ if you haven't already. 
Cheers
